source/nodejs/adaptivecards-site/themes/adaptivecards/source/web.config

```diff
@@ -4,6 +4,7 @@
 		<httpProtocol>
 			<customHeaders>
 				<add name="X-Content-Type-Options" value="nosniff" />
+				<add name="Strict-Transport-Security" value="max-age=63113852; includeSubDomains; preload" />
```

Comment:
Would this have any impact on the PWA portion?

Reply:
I couldn't find any indication that it would make a difference... the short version is that the presence of this header means that requests that _might_ be made via plain HTTP will get transparently upgraded to HTTPS without even attempting to contact the server, so this should theoretically only impact the PWA if it's pulling from HTTP URLs (and even then, it _should_ Just Work™️)